### PR TITLE
Identifying proper values for dataset names and values

### PR DESCRIPTION
**Pull request by tschaffter**
Status: WIP

### CODE REVIEW

**Comment by thomasyu888** [comment]
Looks good to me.  The docker volumes will have to be named:
```
uw_omop_covid_train_05-06-2020
uw_omop_covid_infer_05-06-2020
```

File: infrastructure/get_site.cwl [hunk 1]
```diff
@@ -29,9 +29,9 @@ expression: |
   ${
     if (inputs.evaluation_id == "9614494" || inputs.evaluation_id == "9614451"){
       return {site: "UW",
-              train_dataset_name: "uw_omop_covid",
-              train_dataset_version: "train",
-              infer_dataset_name: "uw_omop_covid",
+              train_dataset_name: "uw_omop_covid_train",
```
Comment (tschaffter) [inline infrastructure/get_site.cwl:32]: Having the keyword `train` in both the variable name and value is not ideal. What about having the value `uw_omop_covid` for both `train_dataset_name` and `infer_dataset_name` AND defining the convention that the training dataset is `${train_dataset_name}_train_${train_dataset_version}`. Idem for the folder name of the inference dataset.

Comment (thomasyu888) [reply infrastructure/get_site.cwl:32]: I thought you wanted to make it easy for sites?  Like in the above scenario, UW can name their docker volume.
```
uw_omop_covid_train
uw_omop_covid_05-06-2020
```

I'm fine with whichever solution we come up with, but I think we need input from Tim

Comment (trberg) [reply infrastructure/get_site.cwl:32]: I think my preference would be to just create volumes like
```
uw_omop_covid_train_05-06-2020
uw_omop_covid_infer_05-06-2020
```
That way, if we end up changing the training data, we can just make a new volume

Comment (thomasyu888) [reply infrastructure/get_site.cwl:32]: Did you want us to tack on "train" or did you think that the train dataset name should just be "uw_omop_covid_train"

**Pull request closed without merging**